embrace-android-sdk/embrace-proguard.cfg

```diff
@@ -8,17 +8,6 @@
 -dontwarn okhttp3.**
 -dontwarn okio.**
 
-## Proguard configuration for Gson
--keepattributes Signature
--keepattributes *Annotation*
--dontwarn sun.misc.**
--keep class com.google.gson.examples.android.model.** { *; }
--keep class * implements com.google.gson.TypeAdapterFactory
--keep class * implements com.google.gson.JsonSerializer
--keep class * implements com.google.gson.JsonDeserializer
--keep class com.google.gson.reflect.TypeToken { *; }
--keep class * extends com.google.gson.reflect.TypeToken
-
 ## Proguard configuration for Arrow
```

Comment:
Nothing needed for Moshi huh? That's nice!

Reply:
Moshi bundles its own proguard rules, and as we use codegen for serialization our adapters are safe.